aws-pricing: add page (#8804)
tldr-pages/tldr: pages/common/aws-pricing.md

# aws pricing

> > Centralized and convenient way to programmatically query Amazon Web Services for services, products, and pricing information.

Reviewer: Perhaps this better
```suggestion
> Query services, products, and pricing information from Amazon Web Services.
```

Author: Much better. Mine was the official description from AWS docs.

> More information: <https://docs.aws.amazon.com/cli/latest/reference/pricing/>.

> - Get a list of service codes to query later for:

Reviewer: ```suggestion
- List service codes of a specific region:
```

`aws pricing describe-services --region {{us-east-1}}`

> - Get a a list of attributes for a given service code:

Reviewer: `get` is more for programming that terminal
```suggestion
- List attributes for a given service code in a specific region:
```

Author: Totally agree!

`aws pricing describe-services --service-code {{AmazonEC2}} --region {{us-east-1}}`

> - Get pricing information for a service code:

Reviewer: ```suggestion
- Print pricing information for a service code in a specific region:
```

`aws pricing get-products --service-code {{AmazonEC2}} --region {{us-east-1}}`

> - Get a list of attribute values to use while filtering:

Reviewer: ```suggestion
- List values for a specific attribute for a service code in a specific region:
```

`aws pricing get-attribute-values --service-code {{AmazonEC2}} --attribute-name {{instanceType}} --region {{us-east-1}}`

> - Get pricing information for a service code and attribute using a filter on the instance type and location:

Reviewer: Which `attribute`? :thinking:
```suggestion
- Print pricing information for a service code and attribute using filters for instance type and location:
```

Author: My fault... It does not make sense, I'm removing this part of the sentence.

Author: Done!

> `aws pricing get-products --service-code {{AmazonEC2}} --filters {{"Type=TERM_MATCH,Field=instanceType,Value=m5.xlarge"}} {{"Type=TERM_MATCH,Field=location,Value=US East (N. Virginia)"}} --region {{us-east-1}}`

Reviewer: Our standard is quotes outside the token syntax
```suggestion
`aws pricing get-products --service-code {{AmazonEC2}} --filters "{{Type=TERM_MATCH,Field=instanceType,Value=m5.xlarge}}" "{{Type=TERM_MATCH,Field=location,Value=US East (N. Virginia)}}" --region {{us-east-1}}`
```

Author: Sorry for that! Will consider it for future contributions...
